example/src/App.js

```diff
@@ -1,10 +1,377 @@
 import React from 'react'
 
-import { ExampleComponent } from '@ibmdotcom/think-banner'
+import { Banner } from '@ibmdotcom/think-banner'
+import { DotcomShell } from '@carbon/ibmdotcom-react'
 import '@ibmdotcom/think-banner/dist/index.css'
+import Content from './Content/Content'
 
 const App = () => {
-  return <ExampleComponent text="IBM Think Banner 😄" />
+  return (
+      <>
+      <Banner />
+        <DotcomShell mastheadProps={{
+          navigation: "default"
+        }}>
+            <Content />
+        </DotcomShell>
+      </>
+      )
 }
 
 export default App
+
+
+// {
```

Comment:
we don't need this commented out section right?

Reply:
Nope, that was just a test object for an example L1, will remove!